js/editor.js

```diff
@@ -19,13 +19,15 @@ jQuery( function( $ ) {
 		const tr = $( this ).closest( 'tr.editor' );
 		const rowId = tr.attr( 'row' );
 		const translation_status = tr.find( '.panel-header' ).find( 'span' ).html();
+		const chatgpt_review_status = JSON.parse( window.localStorage.getItem( 'translate-details-state' ) );
+		const chatgpt_review_enabled = ( 'open' === chatgpt_review_status['details-chatgpt'] );
```

Comment:
Please handle the case when `chatgpt_review_status` is null which can be the case when the user is new or has purged their localStorage.

Reply:
You can check how it was done here: https://github.com/akirk/wordpress.org/blob/trunk/wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/templates/js/editor.js#L297-L298